src/main/java/io/vertx/kafka/client/producer/KafkaWriteStream.java

```diff
@@ -105,8 +105,22 @@
    * @param handler handler called on operation completed
    * @return  current KafkaWriteStream instance
    */
+
   KafkaWriteStream<K, V> write(ProducerRecord<K, V> record, Handler<AsyncResult<RecordMetadata>> handler);
 
+  /*
+  *
+  *
+  * */
```

Comment:
There are these blank lines JavaDoc and following methods don't have methods summary.